test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -596,7 +611,7 @@ def setUp(self):
         a = {'log_events': '4095'}
         self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
         # Configure the scheduler to schedule using vmem
-        a = {'resources': 'ncpus,mem,vmem,host,vnode'}
+        a = {'resources': 'ncpus,mem,vmem,host,vnode,ngpus'}
```

Comment:
I'm thinking we may want to make this part of setUp() for any future GPU/MIC related tests. You should add nmics to the list as well.